### PR TITLE
hdk v0.0.138

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: tests/src/installAgents.ts [hunk 1]
```diff
@@ -1,78 +1,97 @@
...
+const installedAppId = (agentName) => `${agentName}_chat`
 
 export type Memproof = {
-  signed_header: {
-    header: any,
-    signature: Buffer,
-  },
-  entry: any
+	signed_action: {
+		header: any
```
Comment (JettTech) [inline tests/src/installAgents.ts]: Should probably update this to `action` too.
```suggestion
		action: any
```

File: zomes/chat/Cargo.toml [hunk 2]
```diff
@@ -1,37 +1,19 @@
 [package]
-name = "elemental-chat"
+name = "chat"
 authors = ["michael.dougherty@holo.host", "philip.beadle@holo.host", "david.meister@holo.host, tom.gowan@holo.host"]
 version = "0.2.7"
```
Comment (JettTech) [inline zomes/chat/Cargo.toml]: Bump the version.

**Review by JettTech — APPROVED** [review]
Looks good.  Great job with the `TypedPaths` usage.  When you have time for clean-up, I'd recommend renaming variables named with `element` to be renamed with `record` to keep things consistent.

**Comment by zo-el** [comment]
Let's make a release of this PR and I am going to come back to this PR when we have space to clean it up.

**Pull request merged**